docs/trex_events.md

```diff
@@ -20,7 +20,7 @@ the method throws an exception. The following table shows the event types suppor
 | Worksheet | `FilterChanged` , `MarkSelectionChanged` |
 | Parameter | `ParameterChanged` |
 | Settings |  `SettingsChanged` |
-| Dashboard |  None available    |
+| Dashboard |  `DashboardLayoutChanged`    |
```

Comment:
I will also be adding `WorkbookFormattingChanged` based on convo with DashApps Team